examples/rust/xdp/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "xdp"
+version = "0.1.0"
+authors = ["Hengqi Chen <chenhengqi@outlook.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0"
+ctrlc = { version = "3.0", features = ["termination"] }
+libc = "0.2"
+libbpf-rs = { git = "https://github.com/libbpf/libbpf-rs" }
```

Comment:
I've released a version with your XDP patch in -- can you pin the version instead? So that any future API breaking releases don't automatically break these examples as well.